dev/io.openliberty.pages.4.0.internal.factories/src/org/apache/jasper/el/ELResolverImpl.java

```diff
@@ -0,0 +1,129 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.jasper.el;
+
+import java.util.Iterator;
+
+import jakarta.el.ELContext;
+import jakarta.el.ELException;
+import jakarta.el.ELResolver;
+import jakarta.el.ExpressionFactory;
+import jakarta.el.PropertyNotWritableException;
+import jakarta.servlet.jsp.el.VariableResolver;
+
+@Deprecated
```

Comment:
Where did you get this class from? It is marked deprecated.

Reply:
https://github.com/OpenLiberty/open-liberty/blob/2d5a81310c5610a33bc363c4b45ed0210cb1e580/dev/com.ibm.ws.jsp.2.3/src/org/apache/jasper/el/ELResolverImpl.java#L30